io.github.alexays.waybar.yaml

```diff
@@ -0,0 +1,262 @@
+app-id: io.github.alexays.waybar
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.vala
+command: waybar
+finish-args:
+  - --socket=wayland
+  # NOTE: To make the linter happy: Waybar obviously expects Wayland.
+  - --socket=fallback-x11
```

Comment:
no. you need to apply for an exception.

Reply:
Done. https://github.com/flathub-infra/flatpak-builder-lint/pull/596